Readme.md

```diff
@@ -290,6 +290,20 @@ Directories are camel case (ex. `webapp/controlMethods`)
 
 ## Variables
 
+### Use prefixes when naming booleans
+
+When naming booleans, use prefixes that give yes/no answers such as *is*, *has*, and *does*:
```

Comment:
`such as *is*, *are*, *has*` because **_is_** and **_does_** have same job and could cause inconsistency

Reply:
Sorry, I submitted this, but I agree with your comment. Add a new PR and I'll approve it.